### PR TITLE
Add missing labels/annotations

### PR DESCRIPTION
**Pull request by kubasobon**
Towards https://github.com/giantswarm/giantswarm/issues/14537

## Checklist

- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Review by corest — APPROVED** [review]
:partying_face:

File: pkg/generator/generator.go [hunk 1]
```diff
@@ -210,7 +210,14 @@ func (g Generator) GenerateConfig(ctx context.Context, installation, app, ref st
 	name := generateResourceName(app, ref)
 	meta := metav1.ObjectMeta{
 		Name:      name,
-		Namespace: "giantswarm",
+		Namespace: namespace,
+		Labels: map[string]string{
+			"app.kubernetes.io/managed-by": "Helm",
```
Comment (kopiczko) [inline pkg/generator/generator.go:215]: It would be good to also add `"giantswarm.io/managed-by": project.Name()`

**Comment by corest** [comment]
Tested this towards config repo - works fine

File: pkg/generator/generator.go (hunk 1 above)
Comment (kubasobon) [reply pkg/generator/generator.go]: Done.

File: pkg/generator/key/key.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package key
+
+import (
+	"github.com/giantswarm/config-controller/pkg/project"
+)
+
+const (
+	Owner                      = "giantswarm"
```
Comment (kopiczko) [inline pkg/generator/key/key.go:8]: Is it used?

**Pull request merged**